charts/agent-control/charts/agent-control-deployment/templates/_helpers.tpl

```diff
@@ -108,14 +108,14 @@ If you need a list of TODOs, just `grep TODO` on the `values.yaml` and look for
 {{- $k8s := (dict "cluster_name" (include "newrelic.common.cluster" .) "namespace" .Release.Namespace) -}}
 {{- $config = mustMerge $config (dict "k8s" $k8s) -}}
 
-{{- /* Add opamp if enabled */ -}}
-{{- if ((.Values.config).opamp).enabled -}}
-  {{- $opamp := (dict "endpoint" (include "newrelic-agent-control.config.endpoints.opamp" .)) -}}
+{{- /* Add fleet_control if enabled */ -}}
```

Comment:
Before merging I would close [the discussion](https://github.com/newrelic/newrelic-super-agent/pull/943#discussion_r1886447874) in the other PR since some changes might be required